homeassistant/components/ialarm/alarm_control_panel.py

```diff
@@ -0,0 +1,95 @@
+"""Interfaces with iAlarm control panels."""
+import logging
+
+from homeassistant.components.alarm_control_panel import (
+    FORMAT_NUMBER,
+    AlarmControlPanelEntity,
+)
+from homeassistant.components.alarm_control_panel.const import (
+    SUPPORT_ALARM_ARM_AWAY,
+    SUPPORT_ALARM_ARM_HOME,
+)
+from homeassistant.const import CONF_PIN
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DATA_COORDINATOR, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, entry, async_add_entities) -> None:
+    """Set up a iAlarm alarm control panel based on a config entry."""
+    coordinator = hass.data[DOMAIN][entry.entry_id][DATA_COORDINATOR]
+    pin = entry.data.get(CONF_PIN)
+    async_add_entities([IAlarmPanel(coordinator, pin)], False)
```

Comment:
```suggestion
    async_add_entities([IAlarmPanel(coordinator)], False)
```

It doesn't look like the device requires a code.